pkg/dss/geo/s2_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 
 	"github.com/steeling/InterUSS-Platform/pkg/dss/geo/testdata"
-	dspb "github.com/steeling/InterUSS-Platform/pkg/dssproto"
+	dspb "github.com/steeling/InterUSS-Platform/pkg/dss_v1"
```

Comment:
While you are here, you might want to adjust `dspb` -> `dss_v1`.

Reply:
done